packages/gatsby/src/utils/page-data.ts

```diff
@@ -67,6 +69,54 @@ export function pageDataExists(publicDir: string, pagePath: string): boolean {
   return fs.existsSync(getFilePath(publicDir, pagePath))
 }
 
+let lmdbPageQueryResultsCache: GatsbyCacheLmdb
+function getLMDBPageQueryResultsCache(): GatsbyCacheLmdb {
+  if (!lmdbPageQueryResultsCache) {
+    const GatsbyCacheLmdbImpl = require(`./cache-lmdb`).default
+    lmdbPageQueryResultsCache = new GatsbyCacheLmdbImpl({
```

Comment:
I think it's missing an `.init()` here

Reply:
Maybe we can just use `get-cache` utility? https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/utils/get-cache.ts

I know it's being used for plugins but it does pretty much the same thing? Maybe we can use some `internal-tmp-query-results` name for it?

Reply:
> Maybe we can just use get-cache utility?

I don't think we want to use legacy cache implemenation for non-lmdb mode (as that would keep some page query results in memory), so using `get-cache` would look really similar with added extra abstraction? Not sure if worth it?

> I know it's being used for plugins but it does pretty much the same thing? Maybe we can use some internal-tmp-query-results name for it?

👍 for cache name suggestion